DevelopersGuide.adoc

```diff
@@ -2060,7 +2060,7 @@ The program will now look like this:
       (dom/ul
         (map #(ui-person (comp/computed % {:onDelete delete-person})) people)))))
 
-(def ui-person-list (comp/factory PersonList {:keyfn :list/id}))
+(def ui-person-list (comp/factory PersonList))
```

Comment:
Why do you think this change is necessary/right?

Reply:
Yes, I think we should have the :keyfn for the same reason too. However, the code example, as it stands now in the developer guide, does NOT have :keyfn.

There are 2 commits in this PR. The first commit (53ff108) includes 4 edits, the last of which is adding :keyfn. The second commit (160b3d7) reverted the :keyfn addition because I saw in PR #2 (which happened to be about the same :keyfn issue) that you mentioned the following and closed the PR:

"In this case it was intentional. You really only need a keyfn if the item will be used in sequences (via map for example). So, in my mind I would never have a "list of lists", and the component as-written doesn't have a stable ID to use as a key."

So, I added the :keyfn in the first commit (53ff108) and removed it in the second commit (160b3d7). By doing so, I also  wanted to direct attention to the first 3 edits in the first commit which are the gist of this PR. Guess I was overthinking and made this PR confusing with the changes. Hope this clears things up. Thanks again for taking the time to review this.